js/App.js

```diff
@@ -11,13 +11,14 @@ import {
 
 import { theme } from '../config';
 
-import Settings from './scenes/Settings';
-import Profile from './scenes/Profile';
-import Compass from './scenes/Compass';
-import Demos from './scenes/Demos';
+import FTU from './scenes/FTU';
 import Home from './scenes/Home';
 import Item from './scenes/Item';
 import Map from './scenes/Map';
+import Profile from './scenes/Profile';
```

Comment:
The new order reflects the order the routes are declared in the constructor below.

Also I know it's bad to import everything but I wouldn't worry too much as the components are not instantiated.
In the final product, I think it makes sense to be able to access the FTU (remember Firefox OS?)